jekyll/_cci2_ja/faq.md

```diff
@@ -2,150 +2,147 @@
 layout: classic-docs
 title: "よくあるご質問"
 short-title: "よくあるご質問"
-description: "CircleCI 2.0 についてのよくある質問"
-categories: [migration]
+description: "CircleCI 2.0 に関してよく寄せられるご質問"
+categories:
+  - migration
 order: 1
 ---
 
-* 目次
+- 目次
 {:toc}
 
-## 一般
+## 全般
+
+### CircleCI の従業員にプログラム コードを見られるおそれはありませんか?
```

Comment:
is the space intentionally inserted?

Reply:
yeah, it seems they deliberately made the change according to the history on Crowdin.